src/components/ImagesCarousel.js

```diff
@@ -190,7 +190,9 @@ const styles = StyleSheet.create({
     alignSelf: 'center'
   },
   imageContainer: {
-    alignSelf: 'center'
+    alignItems: 'center',
+    alignSelf: 'center',
+    width: '100%'
```

Comment:
does this work for sliders in landscape with more than one image? there was always one centered image and the others images of a slider as "preview" on the left or right.

Reply:
Unfortunately I haven't thought about more than one picture. 😔 We'll see about that when we merge the `epic/SUE` branch with the master.